packages/next/src/build/webpack-config.ts

```diff
@@ -1035,8 +1036,18 @@ export default async function getBaseWebpackConfig(
 
   let hasExternalOtelApiPackage = false
   try {
-    require.resolve('@opentelemetry/api')
-    hasExternalOtelApiPackage = true
+    const opentelemetryPackageJson = require('@opentelemetry/api/package.json')
+    if (opentelemetryPackageJson.version) {
+      // 0.19.0 is the first version of the package that has the `tracer.getSpan` API that we need:
+      // https://github.com/vercel/next.js/issues/48118
+      if (semver.gte(opentelemetryPackageJson.version, '0.19.0')) {
+        hasExternalOtelApiPackage = true
+      } else {
+        Log.error(
```

Comment:
if it errors will this log in all compilers? do we need a conditon to ensure it only log once?